inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp

```diff
@@ -423,6 +444,20 @@ TEST(IEClassBasicTest, smoke_SetConfigHeteroNoThrow) {
     ASSERT_FALSE(value);
 }
 
+TEST_P(IEClassMultiDeviceTestSetConfig, SetConfigMultiDeviceNoThrow) {
```

Comment:
why it's called Multi device? I don't see that `MULTI` is used here :)

Reply:
I think `MULTI` here means test which are run on machines with several GPUs only, but I agree that it should be renamed somehow to avoid confusion.